### DISCUSSION TITLE
Does Freeplane have tagging?

### DISCUSSION THREAD

**Discussion by Foxxsayy** [freeplane/freeplane#1440] [Help]
As in categorical tagging for nodes. I see a bunch of entries for tagging in coding, but I can't find anything about tagging nodes or items in the map.

**Comment by HaaiHenkie** [comment]
Hi, you can use attributes.

**Comment by euu2021** [comment]
For simple tagging, it's possible to use attributes.

For categorical tagging, I see no easy way. One idea is to have a branch called "Tags", with "tag nodes", and then connect the "ordinary nodes" to the "tag nodes". The filters are powerful, so you can filter by tag category. [Mapinsight add-on](https://github.com/adxsoft/MapInsight) can be useful. [This script](https://sourceforge.net/p/freeplane/discussion/758437/thread/00a59ea0d6/) can be useful.
For example:
![image](https://github.com/freeplane/freeplane/assets/77707706/b1b7a5b6-7c9b-4e5e-a75a-b44caf95baa2)

There are some discussions and ideas about tags in the old forum. 
It was the [most voted feature request](https://sourceforge.net/p/freeplane/featurerequests/2179/) in the old forum, although the discussion wasn't thorough.

**Reply by euu2021** [reply]
Also, using the [Teleporter script](https://github.com/freeplane/freeplane/discussions/1011) (set to Connect) can make this tagging easier.
